src/main/java/org/broadinstitute/consent/http/service/ElasticSearchService.java

```diff
@@ -243,30 +256,30 @@ public InstitutionTerm toInstitutionTerm(Institution institution) {
     return new InstitutionTerm(institution.getId(), institution.getName());
   }
 
-  public Response indexDataset(Dataset dataset) throws IOException {
-    return indexDatasetTerms(List.of(toDatasetTerm(dataset)));
+  public Response indexDataset(Dataset dataset, User user) throws IOException {
+    return indexDatasetTerms(List.of(toDatasetTerm(dataset)), user);
```

Comment:
You could remove the call to `toDatasetTerm()` if you delegated to the overloaded method
```suggestion
    return indexDatasets(List.of(dataset), user);
```